### ISSUE TITLE
`env exists?` command

### ISSUE THREAD

**Issue by fdncred** [nushell/engine-q#613]
it would be nice to check for the existence of an environment variable like this `ENV_VAR | env exists?`.

**Comment by jntrnr** [comment]
`"ENV_VAR" in (env).name` is I think the current smallest

**Comment by fdncred** [comment]
agreed, however it's not as intuitive as `env exists?`. I'm wondering if we should add `env exists?` or write a custom command. I tried to write one but couldn't figure it out.

this doesn't work, nor does `$it` in place of `$in`. I'm not sure what the piped input should be named.
```
def "env exists?" [] {
  $in in (env).name
}
```

**Comment by jntrnr** [comment]
There's a limitation with `$in` currently where you can't use it to refer to the custom command, so I went with this workaround:

```
def "env exists?" [] {
   each { $it in ((env).name) }
}
```
